lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -901,6 +918,20 @@ func mergeStates(s1 state_types.StatesByID, s2 state_types.StatesByID) state_typ
 	return s1
 }
 
+func getSubscribersForIMSIs(networkID string, imsis []string, subscriberStorage subscriberstorage.SubscriberStorage) (state_types.StatesByID, error) {
```

Comment:
Maybe a better name:
```suggestion
func getSubscriberStatesForIMSIs(networkID string, imsis []string, subscriberStorage subscriberstorage.SubscriberStorage) (state_types.StatesByID, error) {
```